packages/api/package.json

```
@@ -6,11 +6,11 @@
   "type": "module",
   "main": "dist/worker.js",
   "scripts": {
-    "lt": "npm run lt:cluster & npm run lt:postgrest",
-    "lt:cluster": "npx localtunnel --port 9094 --subdomain \"$(whoami)-cluster-api-nft-storage\"",
-    "lt:postgrest": "npx localtunnel --port 3000 --subdomain \"$(whoami)-postgrest-nft-storage\"",
     "deploy": "wrangler publish --env production",
-    "dev": "wrangler dev",
+    "predev": "./scripts/cli.js db --start && ./scripts/cli.js db-sql --cargo --testing --reset",
+    "dev": "miniflare --watch --debug --env ../../.env",
```

does the "pre" prefix have special meaning in npm? Why not:

```suggestion
    "dev": "run-s dev:**",
    "dev:setup:db": "./scripts/cli.js db --start",
    "dev:setup:sql": "./scripts/cli.js db-sql --cargo --testing --reset",
    "dev:miniflare": "miniflare --watch --debug --env ../../.env",
```

---

yes, it runs before `yarn dev`